common/copycleanuphandler.h

```diff
@@ -0,0 +1,18 @@
+#ifndef COPYCLEANUPHANDLER_H
+#define COPYCLEANUPHANDLER_H
```

Comment:
I noticed it just now as well, is it ok if i add copyright headers all over at the same time?